test/integration/suite_test.go

```diff
@@ -91,6 +91,7 @@ func (s *Suite) SetupTest() {
 	go func() {
 		<-wkrCtx.Done()
 		wrk.Stop()
+		defer wkrQueue.Close()
```

Comment:
It doesn't matter functionally, just pointing it for understanding; you don't need the `defer` keyword, since it's already being called after `wrk.Stop()`. `defer` just means "call this  when the function ends".